### DISCUSSION TITLE
Power Series Neo and keybusinterface?

### DISCUSSION THREAD

**Discussion by luk2009** [taligentx/dscKeybusInterface#322] [dscKeybusInterface Discussion]
Given that DSC Tyco will no longer manufacture the power series panels pc585,1616,1832,1864 and will only continue to produce the power series Neo, is it possible to work with these models?

**Comment by ChrisLizon** [comment]
I am also interested in this. I'm eyeing the HS2032NK panel from a local distributor, but if I can't use this open source integration I'll look for a different product.

**Reply by ChrisLizon** [reply]
Doing some Digging (It's listed in Readme.md, d'oh!)  it seems that the DSC Neo series are using CORBUS not Keybus, and will not be supported. I guess I'm looking for a different product.
